client/src/pages/ThankYouPage.jsx

```diff
@@ -0,0 +1,51 @@
+import "../styles/ThankYouPage.css";
+
+function ThankYouPage() {
+  return (
+    <>
+      <h1 className="thankyou-main-title">Remerciements !</h1>

```

Comment:
I wouldn't add a "!" after "Remerciements" as it is a title

Reply:
ok